src/app/pages/signing/signing.component.ts

```diff
@@ -51,6 +57,12 @@ export class SigningComponent implements OnInit {
     return !this.signingService.signingBoxText || this.sending;
   }
 
+  syncMarkdown(textbox: HTMLTextAreaElement, previewBox: MarkdownComponent) {
+    const scrollPerc: number = (textbox.scrollTop + 160) / textbox.scrollHeight;
```

Comment:
Don't know what "Perc" is, but it's marked as a typo

Reply:
I think `Perc` and `ele` could be renamed to be more clear, @TommyHuang-dev